a10_octavia/contrib/scripts/a10_adc_tree_create

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+if [[ $1 == "delete" ]]; then
+    echo "[-] Deleting member"
+    openstack loadbalancer member delete pool1 mem1
+    echo "[-] Deleting pool"
+    openstack loadbalancer pool delete pool1
+    echo "[-] Deleting listener"
+    openstack loadbalancer listener delete l1
+else
+    openstack loadbalancer listener create --protocol HTTP --protocol-port 8080 --name l1 lb1
```

Comment:
I doubt this script will work, as after loadbalancer create command, the lb will be in "creating" state, untill its completely ready, we can not run next command, it will fail.

Reply:
It does otherwise I wouldn't have bene using. You rerun the script if a failure does occur (which only happens occasionally). We can add waits and such, but it is mainly a script I wrote for myself to make testing easier. It isn't worth commit too much dev time to it since it isn't meant for customers.